projects/plugins/crm/modules/portal/templates/dashboard.php

```diff
@@ -4,58 +4,48 @@
  *
  * This is used as the main dashboard page of the portal
  *
- * @author 		ZeroBSCRM
- * @package 	Templates/Portal/Dashboard
- * @see			https://jetpackcrm.com/kb/
- * @version     3.0
- * 
+ * @package automattic/jetpack-crm
+ * @see     https://jetpackcrm.com/kb/
+ * @version 3.0
  */
 
- 
 defined( 'ABSPATH' ) || exit( 0 ); // Don't allow direct access
 
 global $zbs;
 $portal = $zbs->modules->portal;
 
 do_action( 'zbs_enqueue_scripts_and_styles' );
-$uid = get_current_user_id();
-$uinfo = get_userdata( $uid );
-$cID = zeroBS_getCustomerIDWithEmail($uinfo->user_email);
-
-
 ?>
 
 <div class="alignwide zbs-site-main zbs-portal-grid">
-    <nav class="zbs-portal-nav">
-        <?php
-            $portal->render->portal_nav('dashboard');
-        ?>
-    </nav>
-
-    <div class="zbs-portal-content">
-			<?php 
-				// admin msg (upsell cpp) (checks perms itself, safe to run)
-				$portal->render->admin_message();
-
-				$page_title = __("Welcome to your Dashboard","zero-bs-crm");
-				$page_title = apply_filters('zbs_portal_dashboard_title', $page_title);
-			?>
+	<nav class="zbs-portal-nav">
+		<?php $portal->render->portal_nav( 'dashboard' ); ?>
+	</nav>
+
+	<div class="zbs-portal-content">
+		<?php
+		// admin msg (upsell cpp) (checks perms itself, safe to run)
+		$portal->render->admin_message();
+
+		$page_title = __( 'Welcome to your Dashboard', 'zero-bs-crm' );
+		$page_title = apply_filters( 'zbs_portal_dashboard_title', $page_title );
+		?>
 		<h2><?php echo esc_html( $page_title ); ?></h2>
 		<div class='zbs-entry-content' style="position:relative;">
```

Comment:
Though I realize this line wasn't modified, it also uses a combi of single and double quotes. I'm not sure if the intention is to use double quotes for all class names (see below comment) - where making changes anyway, for consistency. Since this shows up easily on the PR as it's just below some modified lines:
```suggestion
		<div class="zbs-entry-content" style="position:relative;">
```